Automattic/jetpack
# VideoPress: Correct display of VideoPress upsells for security daily plan

This PR correctly displays the Jetpack VideoPress free trial -> purchase upgrade nudge for Security Daily and Security Realtime plans based on when VideoPress was removed from the plan.
 
Mirrors the change made in https://github.com/Automattic/wp-calypso/pull/56870


Fixes # 967-gh-Automattic/greenhouse 

#### Changes proposed in this Pull Request:

* Add a date-based override (Oct 7 2021 00:00 UTC) for existing holders of Jetpack Security plans to NOT show the VideoPress upsell nudge, purchases after that time should see the upsell.
* `moment` added to `webpack.config.json` to allow it to be used in static js builds (help from p1636035969038400/1636034596.033000-slack-CDLH4C1UZ)
* `moment` added as a dev dependency to allow tests to run

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


nope

#### Testing instructions:






1. run `jetpack build plugins/jetpack`
2. Launch a new Jetpack site
2. Connect it to Jetpack and purchase one of the Security Daily plans
3. View Dashboard -> At A Glance ->Scroll to Performance section. You SHOULD see the upsell nudge
4. View Settings -> Performance, you SHOULD see the upsell nudge
5. In mc/store-admin: Change the purchase date of your plan to before Oct 7
6. View (refresh may be necessary) the Settings -> Performance page again, you SHOULD NOT see the upsell nudge
7. View the Dashboard -> At A Glance ->Scroll to Performance section again. You SHOULD NOT see the upsell nudge
8. Repeat the test for the Security Realtime plan
